front/src/Phaser/Game/GameScene.ts

```diff
@@ -1060,26 +1059,26 @@ ${escapedMessage}
         }
     }
 
-    private initPositionFromLayerName(layerName: string) {
+    private initPositionFromLayerName(layerName: string, startLayerName: string | null) {
```

Comment:
The difference between `layerName` and `startLayerName` is not obvious here. Maybe some doc would help?